Automattic/sensei
# Display Enrolled Courses instead of Course Progress

Fixes part of #4958 

### Changes proposed in this Pull Request

* Display Enrolled Courses instead of Course Progress

### Testing instructions

* Create a few courses, add a few students and enroll courses.
* Check if you can see all the enrolments on the Students page.
* Use bulk actions to remove/add some courses for students.
* Check if you see those changes on the Students page.

### Screenshot / Video

https://user-images.githubusercontent.com/329356/164323296-0ac2e63f-9805-4e48-b4d4-fea64ed4e4ad.mp4
